tests/e2e/utils/BrowserTabsUtil.ts

```diff
@@ -0,0 +1,74 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { Key, By, error } from 'selenium-webdriver';
+import { CLASSES } from '../inversify.types';
+import { DriverHelper } from './DriverHelper';
+import { Logger } from './Logger';
+
+@injectable()
+export class BrowserTabsUtil {
```

Comment:
This class is mirroring some behavior from `DriverHelper` and is actually duplicating some of the methods there. 
As the `DriverHelper` class is quite long, I'm +1 for moving all Tabs related functions here - like for example those methods: 

- getCurrentUrl
- waitURL
- navigateAndWaitToUrl
- navigateToUrl
- reloadPage
- maximize

WDYT?

Reply:
The idea is good but doesn't look so easy, I guess a lot of tests should be updated. We can create an issue for that.

Reply:
As I skimmed the usage of those methods, I don't think it's something taking more than half a day/one day of work (changes + some basic testing) so I would prefer to have that done together in one PR. If you're not willing to do that now, then please create the issue.

Reply:
I was thinking more in line of marking the DriverHelper methods as deprecated in this PR and removing them in few released in separate PR.

Reply:
> As I skimmed the usage of those methods, I don't think it's something taking more than half a day/one day of work (changes + some basic testing) so I would prefer to have that done together in one PR. If you're not willing to do that now, then please create the issue.

I strongly disagree with that.
I don't have a day for spending it with such a minor issue.

Reply:
Suggested methods marked as "@deprecated".
Dedicated issue for refactor created: https://github.com/eclipse/che/issues/19484